marklogic-data-hub/src/test/java/com/marklogic/hub/flow/DeleteProvenanceTest.java

```diff
@@ -0,0 +1,52 @@
+package com.marklogic.hub.flow;
+
+import com.marklogic.client.FailedRequestException;
+import com.marklogic.hub.AbstractHubCoreTest;
+import com.marklogic.hub.test.Customer;
+import com.marklogic.hub.test.ReferenceModelProject;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class DeleteProvenanceTest extends AbstractHubCoreTest {
+
+    String provenanceCollection = "http://marklogic.com/provenance-services/record";
+
+    @BeforeEach
+    void installProjectAndRunFlow() {
+        installProjectInFolder("test-projects/provenance-test" );
```

Comment:
We should only do this in the deleteAllProvenanceRecords test - installing the project and running the flow isn't necessary for the other two tests.